erdpy/projects/report/options/size.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+from typing import Optional
+
+from .report_option import ReportOption, str_or_default
+
+
+class Size(ReportOption):
+    def apply(self, wasm_path: Path):
+        size = get_file_size(wasm_path)
+        return str_or_default(size)
```

Comment:
At this point, I'm not very sure why _applying an option called Size_ should return the size of the file.

Perhaps `ReportFeature` instead of `ReportOption` (sorry if I misunderstood). Then, `extract()` instead of `apply()`? (e.g. _we extract a feature and we enrich the report with this information_).